cmd/flags_headers.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/multiformats/go-multiaddr"
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	"github.com/celestiaorg/celestia-node/node"
+)
+
+var (
+	headersTrustedHashF = "headers.trusted-hash"
```

Comment:
I'm guessing `headersTrustedHashF` is meant to stand for `headersTrustedHashFlag`. I'd prefer it not be F and just be `Flag`.

Reply:
Ok. Will change it to Flag